### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Tejas4510**

### CODE REVIEW

**Review by Tejas1045 — DISMISSED** [review]

**Review by kw20210625 — CHANGES_REQUESTED** [review]
Change something

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# terraform_module
+#reviewing changes
```
Comment (Tejas1045) [inline README.md:2]: change the text

**Review by Tejas1045 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Tejas1045) [inline README.md:2]: resolve it

**Review by Tejas1045 — CHANGES_REQUESTED** [review]

**Review by Tejas1045 — APPROVED** [review]

**Review by kw20210625 — APPROVED** [review]
approved

**Pull request merged**